stablehlo/dialect/StablehloOps.td

```diff
@@ -304,8 +304,9 @@ def StableHLO_CosineOp: StableHLO_UnaryElementwiseOp<"cosine",
 }
 
 def StableHLO_ExpOp: StableHLO_UnaryElementwiseOp<"exponential",
-    [Pure, HLO_CompatibleOperandsAndResultType], HLO_FpOrComplexTensor> {
-  let summary = "Exponential operator";
+    [Pure, HLO_CompatibleOperandsAndResultType /* exponential_c1 */],
```

Comment:
Are we using the `/*foo_c1*/` or `/* foo_c1 */` convention? I think the checklist proposes the former.

Reply:
Ah sure, updated in this PR and also other PRs as well.